### PR TITLE
Update Donations Page

### PR DESCRIPTION
**Pull request by pzalep1**
This PR updates the donations page to include the new corporate giving level. 
screencapture-localhost-4200-donate-2020-08-12-13_59_54

### CODE REVIEW

File: src/app/cube/donate/donate.component.html [hunk 1]
```diff
@@ -2,25 +2,50 @@
   <section class="hero">
       <div class="inner">
           <h1 class="header">Donate To CLARK</h1>
-          <p class="description">CLARK is a digital library that hosts a diverse collection of free cybersecurity Learning Objects. In order to maintain our system and continue to provide our Learning Objects at no cost we need your help.</p>
+          <p class="description">The CLARK Platform is developed and maintained by TU students and relies on government and industry funding for support.</p>
```
Comment (Olaniyan99) [inline src/app/cube/donate/donate.component.html:5]: might wanna change the wordings a little, instead of "industry funding" to "industrial funding".

Comment (Olaniyan99) [reply src/app/cube/donate/donate.component.html:5]: everything else looks good

Comment (pzalep1) [reply src/app/cube/donate/donate.component.html:5]: Industrial funding actually refers to a specific fee for a tax form I believe.

Comment (pzalep1) [reply src/app/cube/donate/donate.component.html:5]: I've been spending a little too much time with accountants

**Pull request merged**